app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -249,6 +249,18 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
+    var showCollectionsPlaceholderOnHome by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_show_collections_placeholder_home),
+        default = true
+    )
+
+    fun dismissCollectionsPlaceholderOnHome() {
```

Comment:
Is the extra function needed? Wouldn't `showCollectionsPlaceholderOnHome = false` work?